foundations/javascript_basics/fundamentals-3.md

```diff
@@ -40,7 +40,7 @@ For now, just write each function and test the output with `console.log`.
 
 1. Write a function called `add7` that takes one number and returns that number + 7.
 2. Write a function called `multiply` that takes 2 numbers and returns their product.
-3. Write a function called `capitalize` that takes a string and returns that string with _only_ the first letter capitalized.  Make sure that it can take strings that are lowercase, UPPERCASE or BoTh.
+3. Write a function called `capitalize` that takes a string and returns that string with _only_ the first letter capitalized.  Make sure that it can take strings that are lowercase, UPPERCASE or both.
```

Comment:
BoTh is intentional. It is used as an example of a word which has both lowercase and uppercase letters. 

Anyway, thanks for wanting to contribute! Closing this pull request now.